frontend/src/components/Uploader/ProvidersUplodader.tsx

```diff
@@ -58,14 +58,15 @@ const COLUMNS = [
   { aliases: ['Latitude'] },
   { aliases: ['Longitude'] },
   { aliases: ['NPI'] },
+  { aliases: ['Name'] },
```

Comment:
First Name and Last Name as separate columns are more common, right? We should add a function to smoosh them together like Languages

Reply:
Done